include/yaml-cpp/emittermanip.h

```diff
@@ -36,6 +37,10 @@ enum EMITTER_MANIP {
   LongBool,       // yes, On
   ShortBool,      // y, t
 
+  // null manipulators
+  NullAsTilde,     // ~
```

Comment:
I think I prefer these as

TildeNull
VerboseNull (or maybe JsonNull?)

which match the bool manipulators all ending in Bool. Open to discussion, though.

Reply:
Done (I went for `JsonNull`).